codecov.yml

```diff
@@ -1,4 +1,6 @@
 # Repository Yaml
+codecov:
+  token: 684bc22e-c6ca-4ac5-b5be-d1dd250a4567
```

Comment:
... committing this seems vaguely questionable, but I'm not a circle/codecov guru - don't know how else to do it.

Reply:
added to environment